expression/builtin_math_vec.go

```diff
@@ -658,11 +659,30 @@ func (b *builtinRoundWithFracIntSig) vectorized() bool {
 	return true
 }
 func (b *builtinCRC32Sig) vectorized() bool {
-	return false
+	return true
 }
 
 func (b *builtinCRC32Sig) vecEvalInt(input *chunk.Chunk, result *chunk.Column) error {
-	return errors.Errorf("not implemented")
+	n := input.NumRows()
+	buf, err := b.bufAllocator.get(types.ETString, n)
+	if err != nil {
+		return err
+	}
+	defer b.bufAllocator.put(buf)
+	if err := b.args[0].VecEvalString(b.ctx, input, buf); err != nil {
+		return err
+	}
+	result.ResizeInt64(n, false)
+	i64s := result.Int64s()
+	for i := range i64s {
+		if buf.IsNull(i) {
+			i64s[i] = 0
```

Comment:
Don't need to set `i64s[i]=0`. If buf.IsNull(i) is true, the value is meaningless.

Reply:
And I guess the `ResizeInt64` will initial all the value to 0? so this line is indeed redundant. I will remove it.